src/components/NavBar/NavbarLinks.jsx

```diff
@@ -1,5 +1,5 @@
 // React Imports
-import { React, useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
Not sure if I've asked this yet, but why does this codebase/project seem to require `import React from 'react'`?

This hasn't been needed since v17.   

Is there something about vite or the linting process for this project?

Reply:
Personally, I wasn't aware this was updated, so that's good to know.

However, at least my linter throws errors when I don't import it.

I'll see if I can figure out whatever's wrong there.

Reply:
For this, I'm basically following the linter. It seems to complain when you try to make .jsx files without it.